app/models/work_skin.rb

```diff
@@ -31,6 +31,7 @@ def self.import_basic_formatting
     File.open(Rails.public_path + '/images/skins/previews/basic_formatting.png', 'rb') {|preview_file| skin.icon = preview_file}
     skin.official = true
     skin.save!
+    Rails.cache.increment('skins_generation/'+skin.type.nil? ? ("site_skin") : skin.id.to_s )
```

Comment:
Surrounding space missing for operator `+`.<br>Space inside parentheses detected.